src/containers/UserProfile/UserProfile.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect } from "react";
+import "./UserProfile.css";
+import db from "../../firebase";
+
+function UserProfile() {
+  // eslint-disable-next-line
+  const [edit, setEdit] = useState(false);
+
+  function handleClick() {
+    setEdit((edit) => !edit);
+  }
+
+  //needs work
+  function handleSubmit() {
+    console.log("Data Edited");
+  }
+
+  const [user, setUser] = useState([]);
+
+  const fetchUser = async () => {
+    const response = db.collection("users");
+    const data = await response.get();
+    data.docs.forEach((item) => {
```

Comment:
I dont think we will be fetching all the users there will be only one user logged in